css/src/tokens/index.scss

```diff
@@ -1,5 +1,6 @@
 @import './palette.scss';
 @import './animations.scss';
+@import './breakpoints.scss';
```

Comment:
You need to rebase ! this wont be required

Reply:
Rebased!